workflow/setup_expt.py

```diff
@@ -186,6 +187,19 @@ def input_args(*argv):
 
     ufs_apps = ['ATM', 'ATMA', 'ATMW', 'S2S', 'S2SA', 'S2SW', 'S2SWA']
 
+    def _validate_interval(interval_str):
+        err_msg = f'must be a non-negative integer multiple of 6 ({interval_str} given)'
```

Comment:
Should it always be a multiple of 6 or should it be a multiple of `assim_freq` for cycled runs?  I'm not sure if anyone changes `assim_freq`, so it may be a moot question.

Reply:
`assim_freq` (see comment below), but I don't think there is a way to know it at this point, so I had to hard-code it.